frontend/src/hooks/offramp/useSubmitRamp.ts

```diff
@@ -82,10 +83,18 @@ export const useSubmitRamp = () => {
             if (!response.ok) {
               // Response can also fail due to invalid KYC. Nevertheless, this should never be the case, as when we create the user we wait for the KYC
               // to be valid, or retry.
-              if (response.status === 404) {
+              if (response.status === 404 && !isValidCnpj(taxId) && isValidCpf(taxId)) {
```

Comment:
It should be enough to only check for a valid CPF. The same input cannot be a valid CPF and CNPJ.

```suggestion
              if (response.status === 404 && isValidCpf(taxId)) {
```